src/HealthChecks.UI/Configuration/Settings.cs

```diff
@@ -27,12 +27,10 @@ public Settings AddHealthCheckEndpoint(string name, string uri)
         
         public Settings AddWebhookNotification(string name, string uri, string payload, string restorePayload = "")
         {
-            if (!Uri.TryCreate(uri, UriKind.Absolute, out Uri absoluteUri)) throw new ArgumentException($"Invalid uri: {uri}");
```

Comment:
This check is removed as Uris processing will come later when parsing relative / absolute urls

Reply:
@Tratcher @jkotalik @rynowak this is the part of the PR I am most concerned about as I do not know if they is another way of waiting until the listening urls are reported by the IServerAddressesFeature when using IIS with InProcess hosting model. 

Without this knowledge, I can just delay the healthcheck collection execution until the ServerAddressesService knows this listerning addresses.

Could you give some advice?